templates/cisco_xr_show_bgp_neighbors.template

```diff
@@ -32,7 +34,7 @@ Value LOCAL_PORT (\d+)
 Value REMOTE_ADDRESS (.*)
 Value REMOTE_PORT (\d+)
 Value LAST_RESET (\S+)
-Value LAST_RESET_REASON (.+?)
+Value LAST_RESET_REASON (.*)
```

Comment:
Was this not working for your data? Follows for line 106

Reply:
Without ending the line with `\s*$$` that will happen, but my update included `\s*$$`, so it will capture everything up to the end of the line.

Reply:
@jmcgill298   I see what the issue was.  Some routers output 
```  ^\s+Last\s+reset\s+${LAST_RESET}\,\s+due\s+to\s+${LAST_RESET_REASON}```
and some just output
```  ^\s+Last\s+reset\s+${LAST_RESET}```

So there is a need to match either or (or just omit the values) .  so the .+? failed with the first output scenario...  I'll look into a more proper solution.

Reply:
@jmcgill298   Can you check out the latest commit 60e6e56 and see if that makes sense?